src/services/celebration-service.ts

```diff
@@ -56,7 +56,7 @@ export class CelebrationService {
 
                     // Get the blacklist data for this guild
                     let blacklistData = guildCelebrationData.blacklistedMembers.map(
-                        m => m.UserDiscordId
+                        m => m.DiscordId
```

Comment:
Similar to above, maybe rename `m` to `b`?

Reply:
Done